<html><head><meta http-equiv="content-type" content="text/html; charset=UTF-8"><title>Conversation with i_bratchuk@wargaming.net at Срд 19 Ліс 2014 13:53:55 on r_verenich@wargaming.net/ (jabber)</title></head><body><h3>Conversation with i_bratchuk@wargaming.net at Срд 19 Ліс 2014 13:53:55 on r_verenich@wargaming.net/ (jabber)</h3>
<font color="#A82F2F"><font size="2">(13:53:55)</font> <b>Igor Bratchuk:</b></font> <html xmlns='http://jabber.org/protocol/xhtml-im'><body xmlns='http://www.w3.org/1999/xhtml' style='font-size: 9pt; text-decoration: ; font-family: arial; font-weight: normal; color: #004200; font-style: normal'>привет</body></html><br/>
<font color="#A82F2F"><font size="2">(13:54:13)</font> <b>Igor Bratchuk:</b></font> <html xmlns='http://jabber.org/protocol/xhtml-im'><body xmlns='http://www.w3.org/1999/xhtml' style='font-size: 9pt; text-decoration: ; font-family: arial; font-weight: normal; color: #004200; font-style: normal'>подскажи плз  <br/>при cluster-update ведь происходит обновление liquibase ? <br/>вопрос связан с тем, что DBA добавили в рамках https://jira.wargaming.net/browse/WOTDB-1318 изменения в базу <br/>и они у меня что-то не применяются при апе</body></html><br/>
<font color="#16569E"><font size="2">(13:59:58)</font> <b>Roman Verenich:</b></font> привет!<br/>
<font color="#16569E"><font size="2">(14:00:14)</font> <b>Roman Verenich:</b></font> да, liquibase проходит при апдейте<br/>
<font color="#A82F2F"><font size="2">(14:00:38)</font> <b>Igor Bratchuk:</b></font> <html xmlns='http://jabber.org/protocol/xhtml-im'><body xmlns='http://www.w3.org/1999/xhtml' style='font-size: 9pt; text-decoration: ; font-family: arial; font-weight: normal; color: #004200; font-style: normal'>мне уже Петрович помог, спс</body></html><br/>
<font color="#A82F2F"><font size="2">(14:00:42)</font> <b>Igor Bratchuk:</b></font> <html xmlns='http://jabber.org/protocol/xhtml-im'><body xmlns='http://www.w3.org/1999/xhtml' style='font-size: 9pt; text-decoration: ; font-family: arial; font-weight: normal; color: #004200; font-style: normal'>спс</body></html><br/>
<font color="#A82F2F"><font size="2">(14:00:43)</font> <b>Igor Bratchuk:</b></font> <html xmlns='http://jabber.org/protocol/xhtml-im'><body xmlns='http://www.w3.org/1999/xhtml' style='font-size: 9pt; text-decoration: ; font-family: arial; font-weight: normal; color: #004200; font-style: normal'> )</body></html><br/>
<font color="#16569E"><font size="2">(14:00:48)</font> <b>Roman Verenich:</b></font> а что было?<br/>
<font color="#A82F2F"><font size="2">(14:01:22)</font> <b>Igor Bratchuk:</b></font> <html xmlns='http://jabber.org/protocol/xhtml-im'><body xmlns='http://www.w3.org/1999/xhtml' style='font-size: 9pt; text-decoration: ; font-family: arial; font-weight: normal; color: #004200; font-style: normal'>ликвидбейз не обновляется при апе </body></html><br/>
<font color="#A82F2F"><font size="2">(14:01:26)</font> <b>Igor Bratchuk:</b></font> <html xmlns='http://jabber.org/protocol/xhtml-im'><body xmlns='http://www.w3.org/1999/xhtml' style='font-size: 9pt; text-decoration: ; font-family: arial; font-weight: normal; color: #004200; font-style: normal'>раз в сутки только</body></html><br/>
<font color="#A82F2F"><font size="2">(14:01:34)</font> <b>Igor Bratchuk:</b></font> <html xmlns='http://jabber.org/protocol/xhtml-im'><body xmlns='http://www.w3.org/1999/xhtml' style='font-size: 9pt; text-decoration: ; font-family: arial; font-weight: normal; color: #004200; font-style: normal'>нужно было через update-opt делать</body></html><br/>
<font color="#A82F2F"><font size="2">(14:01:55)</font> <b>Igor Bratchuk:</b></font> <html xmlns='http://jabber.org/protocol/xhtml-im'><body xmlns='http://www.w3.org/1999/xhtml' style='font-size: 9pt; text-decoration: ; font-family: arial; font-weight: normal; color: #004200; font-style: normal'>а сам баг заключается в ошибке на сервере</body></html><br/>
<font color="#A82F2F"><font size="2">(14:02:00)</font> <b>Igor Bratchuk:</b></font> <html xmlns='http://jabber.org/protocol/xhtml-im'><body xmlns='http://www.w3.org/1999/xhtml' style='font-size: 9pt; text-decoration: ; font-family: arial; font-weight: normal; color: #004200; font-style: normal'>Wed 19 Nov 2014 10:39:03.695  by1-dp-vm35 BaseApp    Script INFO     [EXCEPTION] (server_common/database/_InsertTask.py, 61): <br/>Wed 19 Nov 2014 10:39:03.695  by1-dp-vm35 BaseApp    Script ERROR    Traceback (most recent call last): <br/>Wed 19 Nov 2014 10:39:03.695  by1-dp-vm35 BaseApp    Script ERROR      File &quot;/home/wotqac3500/bw/res/wot/scripts/server_common/database/_InsertTask.py&quot;, line 39, in doBackgroundTask <br/>Wed 19 Nov 2014 10:39:03.696  by1-dp-vm35 BaseApp    Script ERROR        cu.execute(queryInfo[&apos;query&apos;], paramsToTuple(self._params[insAction], queryInfo.get(&apos;positionsMapping&apos;, {}), queryInfo.get(&apos;transformers&apos;, {}))) <br/>Wed 19 Nov 2014 10:39:03.696  by1-dp-vm35 BaseApp    Script ERROR      File &quot;cursor.pyx&quot;, line 120, in oursql.Cursor.execute (oursqlx/oursql.c:20423) <br/>Wed 19 Nov 2014 10:39:03.696  by1-dp-vm35 BaseApp    Script ERROR      File &quot;cursor.pyx&quot;, line 111, in oursql.Cursor.execute (oursqlx/oursql.c:20285) <br/>Wed 19 Nov 2014 10:39:03.696  by1-dp-vm35 BaseApp    Script ERROR      File &quot;statement.pyx&quot;, line 157, in oursql._Statement.prepare (oursqlx/oursql.c:10414) <br/>Wed 19 Nov 2014 10:39:03.697  by1-dp-vm35 BaseApp    Script ERROR      File &quot;statement.pyx&quot;, line 127, in oursql._Statement._raise_error (oursqlx/oursql.c:9938) <br/>Wed 19 Nov 2014 10:39:03.697  by1-dp-vm35 BaseApp    Script ERROR    ProgrammingError: (1054, &quot;Unknown column &apos;clan_motto&apos; in &apos;field list&apos;&quot;, None) <br/>Wed 19 Nov 2014 10:39:03.702  by1-dp-vm35 BaseApp    Script INFO     [ERROR] (server_common/database/_InsertTask.py, 58): action: ins_fort_public_info <br/>Wed 19 Nov 2014 10:39:03.702  by1-dp-vm35 BaseApp    Script INFO     [ERROR] (server_common/database/_InsertTask.py, 59): params: {&apos;ins_fort_public_info&apos;: {&apos;vacation_start&apos;: datetime.datetime(1970, 1, 1, 0, 0), &apos;avg_building_level10&apos;: 68, &apos;def_hour_change_day&apos;: None, &apos;battle_count_for_fort&apos;: 0, &apos;vacation_finish&apos;: datetime.datetime(1970, 1, 1, 0, 0), &apos;next_start_def_hour&apos;: None, &apos;home_periphery_id&apos;: 3501, &apos;clan_description&apos;: &apos;descr&apos;, &apos;fort_level&apos;: 8, &apos;off_day&apos;: -1, &apos;finish_def_hour&apos;: 12, &apos;hidden&apos;: 0, &apos;clan_motto&apos;: &apos;motto&apos;, &apos;next_finish_def_hour&apos;: None, &apos;profit_factor10&apos;: 0, &apos;clan_name&apos;: &apos;Tttn&apos;, &apos;next_vacation_finish&apos;: datetime.datetime(1970, 1, 1, 0, 0), &apos;next_vacation_start&apos;: datetime.datetime(1970, 1, 1, 0, 0), &apos;clan_abbrev&apos;: &apos;TTTN&apos;, &apos;next_off_day&apos;: None, &apos;off_day_change_day&apos;: None, &apos;start_def_hour&apos;: 11, &apos;clan_db_id&apos;: 46}} <br/>Wed 19 Nov 2014 10:39:03.702  by1-dp-vm35 BaseApp    Script INFO     [ERROR] (server_common/database/_InsertTask.py, 60): query: INSERT INTO `wgFortPublicInfo` (`clan_db_id`, `clan_abbrev`, `clan_name`, `clan_motto`, `clan_description`, `vacation_start`, `vacation_finish`, `start_def_hour`, `finish_def_hour`, `off_day`, `home_periphery_id`, `fort_level`, `avg_building_level10`, `profit_factor10`, `battle_count_for_fort`, `next_vacation_start`, `next_vacation_finish`, `next_start_def_hour`, `next_finish_def_hour`, `next_off_day`, `def_hour_change_day`, `off_day_change_day`, `hidden`) VALUES (?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?, ?) on duplicate key update clan_db_id=VALUES(clan_db_id), clan_abbrev=VALUES(clan_abbrev), clan_name=VALUES(clan_name), clan_motto=VALUES(clan_motto), clan_description=VALUES(clan_description), vacation_start=VALUES(vacation_start), vacation_finish=VALUES(vacation_finish), start_def_hour=VALUES(start_def_hour), finish_def_hour=VALUES(finish_def_hour), off_day=VALUES(off_day), home_periphery_id=VALUES(home_periphery_id), fort_level=VALUES(fort_level), avg_building_level10=VALUES(avg_building_level10), profit_factor10=VALUES(profit_factor10), battle_count_for_fort=VALUES(battle_count_for_fort), next_vacation_start=VALUES(next_vacation_start), next_vacation_finish=VALUES(next_vacation_finish), next_start_def_hour=VALUES(next_start_def_hour), next_finish_def_hour=VALUES(next_finish_def_hour), next_off_day=VALUES(next_off_day), def_hour_change_day=VALUES(def_hour_change_day), off_day_change_day=VALUES(off_day_change_day), hidden=VALUES(hidden) <br/>Wed 19 Nov 2014 10:39:03.702  by1-dp-vm35 BaseApp    Script INFO     [EXCEPTION] (server_common/database/_InsertTask.py, 61): <br/>Wed 19 Nov 2014 10:39:03.702  by1-dp-vm35 BaseApp    Script ERROR    Traceback (most recent call last): <br/>Wed 19 Nov 2014 10:39:03.702  by1-dp-vm35 BaseApp    Script ERROR      File &quot;/home/wotqac3500/bw/res/wot/scripts/server_common/database/_InsertTask.py&quot;, line 39, in doBackgroundTask <br/>Wed 19 Nov 2014 10:39:03.702  by1-dp-vm35 BaseApp    Script ERROR        cu.execute(queryInfo[&apos;query&apos;], paramsToTuple(self._params[insAction], queryInfo.get(&apos;positionsMapping&apos;, {}), queryInfo.get(&apos;transformers&apos;, {}))) <br/>Wed 19 Nov 2014 10:39:03.702  by1-dp-vm35 BaseApp    Script ERROR      File &quot;cursor.pyx&quot;, line 120, in oursql.Cursor.execute (oursqlx/oursql.c:20423) <br/>Wed 19 Nov 2014 10:39:03.703  by1-dp-vm35 BaseApp    Script ERROR      File &quot;cursor.pyx&quot;, line 111, in oursql.Cursor.execute (oursqlx/oursql.c:20285) <br/>Wed 19 Nov 2014 10:39:03.703  by1-dp-vm35 BaseApp    Script ERROR      File &quot;statement.pyx&quot;, line 157, in oursql._Statement.prepare (oursqlx/oursql.c:10414) <br/>Wed 19 Nov 2014 10:39:03.703  by1-dp-vm35 BaseApp    Script ERROR      File &quot;statement.pyx&quot;, line 127, in oursql._Statement._raise_error (oursqlx/oursql.c:9938) <br/>Wed 19 Nov 2014 10:39:03.704  by1-dp-vm35 BaseApp    Script ERROR    ProgrammingError: (1054, &quot;Unknown column &apos;clan_motto&apos; in &apos;field list&apos;&quot;, None)</body></html><br/>
<font size="2">(18:25:50)</font><b> Igor Bratchuk has left the conversation.</b><br/>
